source/includes/mongodb-compatibility-table-java.rst

```diff
@@ -0,0 +1,159 @@
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+   :class: compatibility-large
+
+   * - Java Driver Version
+     - MongoDB 4.2
+     - MongoDB 4.0
+     - MongoDB 3.6
+     - MongoDB 3.4
+     - MongoDB 3.2
+     - MongoDB 3.0
+     - MongoDB 2.6
+
+   * - Version 3.12
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.11
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.10
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.9
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.8
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.7
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.6
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.5
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.4
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.3
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.2
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+     - ✓
+
+   * - Version 3.1
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - Version 3.0
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - Version 2.14
+     -
+     -
+     -
+     -
+     - ✓ [*]_
+     - ✓
+     - ✓
+
+   * - Version 2.13
+     -
+     -
+     -
+     -
+     -
+     - ✓
+     - ✓
+
+   * - Version 2.12
+     -
+     -
+     -
+     -
+     -
+     -
+     - ✓
+
+.. [*] The 2.14 driver does not support all MongoDB 3.2 features (e.g. read concern); however, if you are currently on a version 2.x driver and would like to run against MongoDB 3.2 but cannot upgrade to driver version 3.2, use the 2.14 driver.
```

Comment:
I think it's time to stop listing 2.x drivers here.

Reply:
Should 2.x versions of the driver also be removed from the Language Compatibility table?

Reply:
Remove 2.x everywhere IMO